templates/docs/examples/patterns/forms/fieldset-required.html

```diff
@@ -0,0 +1,24 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Forms / Fieldset required{% endblock %}
+
+{% block standalone_css %}patterns_forms{% endblock %}
+
+{% block content %}
+<form>
+  <fieldset>
+    <legend class="is-required" for="list-input-1">Required input</legend>
+    <label class="p-radio">
+      <input type="radio" class="p-radio__input" name="radioPattern" aria-labelledby="radioExample1">
```

Comment:
These three `span`s all have the same `id`, and the `inputs` have the same `aria-labelledby` - can we adjust so the `id` fields are unique and `aria-labelledby` references the ID of the adjacent label?